README.md

```diff
@@ -6,20 +6,73 @@ This is a Kubernetes Operator (https://coreos.com/operators/) which deploys Mong
 
 This codebase is currently _pre-alpha_, and is not ready for use.
 
-If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you may be looking for the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
+If you are a MongoDB Enterprise customer, or need Enterprise features such as Backup, you can use the [MongoDB Enterprise Operator for Kubernetes](https://github.com/mongodb/mongodb-enterprise-kubernetes).
 
+## Installation
 
+### Prerequisites
 
+Before you install the MongoDB Community Kubernetes Operator, you must:
 
-## Licensing
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+2. Install [kind](https://kind.sigs.k8s.io/)
```

Comment:
technically `kind` isn't a prerequisite, just having any running Kubernetes cluster is. Kind is just an easy way get one up and running on your own machine. What do you think about changing this to be more generic along the lines of `have access to a kubernetes cluster` and including a recommendation for kind with the relevant link?

Reply:
We discussed this with @chatton.

We should say that "Kubernetes" is a requirement, any flavor; but we use `Kind` for testing, and that's our recommended testing Kubernetes flavor.

Reply:
Updated.